eclair-node/src/test/scala/fr/acinq/eclair/api/ApiServiceSpec.scala

```diff
@@ -145,11 +141,12 @@ class ApiServiceSpec extends FunSuite with ScalatestRouteTest with IdiomaticMock
 
   test("'usablebalances' asks router for current usable balances") {
 
+    val remoteNodeId = PublicKey(hex"03af0ed6052cf28d670665549bc86f4b721c9fdb309d40c58f5811f63966e005d0")
```

Comment:
nit: this is `aliceNodeId` right?

Reply:
Good catch thanks :) , fixed in 3469eb5 (for some reason github is not showing the last commit here on the PR)